src/server/worker/pipeline/transform/registry.go

```diff
@@ -632,14 +616,9 @@ func (reg *registry) processJobEgressing(ppj *pendingPipelineJob) error {
 func failedInputs(pachClient *client.APIClient, pipelineJobInfo *pps.PipelineJobInfo) ([]string, error) {
 	var failed []string
 	blockCommit := func(name string, commit *pfs.Commit) error {
-		ci, err := pachClient.PfsAPIClient.InspectCommit(pachClient.Ctx(),
-			&pfs.InspectCommitRequest{
-				Commit:     commit,
-				BlockState: pfs.CommitState_FINISHED,
-			})
+		ci, err := pachClient.BlockCommit(commit.Branch.Repo.Name, commit.Branch.Name, commit.ID)
```

Comment:
I guess you could make `blockCommit` take strings, since that's what the input has, anyway (unless you're anticipating me changing `BlockCommit` to take a commit proto)

Reply:
I am planning to do a pass over all the methods in `client/pfs.go` and `client.pps.go` to make them all take `Repo`, `Branch`, and `Commit` protos, but after this lands in `master`.